pipelines/broad/arrays/imputation_beagle/ImputationBeagle.wdl

```diff
@@ -1,11 +1,12 @@
 version 1.0
 
+import "../../../../structs/imputation/ImputationBeagleStructs.wdl" as structs
 import "../../../../tasks/broad/ImputationTasks.wdl" as tasks
 import "../../../../tasks/broad/ImputationBeagleTasks.wdl" as beagleTasks
 
 workflow ImputationBeagle {
 
-  String pipeline_version = "0.0.1"
+  String pipeline_version = "1.0.0"
```

Comment:
oooo we ready for 1.0.0 now?!?!?!

Reply:
once we merge to develop we will! this is kinda meaningless till then though